checks.d/process.py

```diff
@@ -308,3 +314,4 @@ def _process_service_check(self, name, nb_procs, bounds):
             tags=tag,
             message=message_str % (status_str[status], nb_procs, name)
         )
+
```

Comment:
This blank line is making Flake8 complain: https://travis-ci.org/DataDog/dd-agent/jobs/99521898#L270

@handigarde could you remove it? :)